app/templates/package.json

```diff
@@ -45,13 +50,13 @@
     "gulp-plumber": "^0.6.6",
     "gulp-rename": "^1.2.0",
     "gulp-sourcemaps": "^1.3.0",
-    "gulp-uglifyjs": "^0.5.0",
-    "isparta": "^1.0.1",
-    "jshint-stylish": "^1.0.0",
+    "gulp-uglify": "^1.2.0",
+    "isparta": "^2.2.0",
+    "mkdirp": "^0.5.0",
     "mocha": "^2.1.0",
     "run-sequence": "^1.0.2",
     "sinon": "^1.12.2",
-    "sinon-chai": "^2.6.0",
+    "sinon-chai": "^2.7.0",
     "vinyl-source-stream": "^1.0.0"
```

Comment:
Theres a number of other changes (eslint) that also need to be merged in with this PR.

@jmeas can you help me out with this?